pulp_file/app/tasks/acs.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from datetime import datetime
+
+from pulpcore.plugin.models import (
+    AlternateContentSource,
+    AlternateContentSourcePath,
+    ProgressReport,
+    TaskGroup,
+)
+from pulpcore.plugin.tasking import dispatch
+
+from pulp_file.app.tasks import synchronize
+from pulp_file.app.models import FileRepository
+
+metadata_files = []
+
+
+def file_acs_refresh(acs_pk):
+    """
+    Refresh alternative content sources.
+    """
+    acs = AlternateContentSource.objects.get(pk=acs_pk)
+    acs_paths = AlternateContentSourcePath.objects.filter(alternate_content_source=acs_pk)
+    with ProgressReport(
+        message="Refreshing ACS metadata", code="acs.refreshing.metadata", total=len(acs_paths)
```

Comment:
This should likely identify that it's refreshing N repos.
```suggestion
        message="Dispatching tasks to refresh metadata", code="acs.refresh.dispatch", total=len(acs_paths)
```